packages/cfd/src/Containers/dmt5-trade-modal.tsx

```diff
@@ -191,121 +176,12 @@ const DMT5TradeModal = observer(
-                            type='button'
-                            href={getPlatformMt5DownloadLink('linux')}
-                            target='_blank'
-                            rel='noopener noreferrer'
-                        >
-                            <Text size='xxs' weight='bold' color='prominent'>
-                                {localize('Learn more')}
-                            </Text>
-                        </a>
-                    </div>
-                </div>
-                <Text
-                    align='center'
-                    as='div'
-                    className='cfd-trade-modal__download-center-text'
-                    size={is_mobile ? 'xxxs' : 'xxs'}
-                    weight='bold'
-                >
-                    {localize(
-                        'Download {{ platform }} on your phone to trade with the {{ platform }} {{ account }} account',
-                        {
-                            platform: getCFDPlatformLabel(CFD_PLATFORMS.MT5),
-                            account: getAccountTitle(),
-                        }
-                    )}
-                </Text>
-                <div className='cfd-trade-modal__download-center-options'>
-                    <div className='cfd-trade-modal__download-center-options--mobile-links'>
-                        <a
-                            href={mt5_trade_account?.white_label?.download_links?.ios}
-                            target='_blank'
-                            rel='noopener noreferrer'
-                        >
-                            <Icon icon='IcInstallationApple' width={135} height={40} />
-                        </a>
-                        <a
-                            href={mt5_trade_account?.white_label?.download_links?.android}
-                            target='_blank'
-                            rel='noopener noreferrer'
-                        >
-                            <Icon icon='IcInstallationGoogle' width={135} height={40} />
-                        </a>
-                        <a href={getPlatformMt5DownloadLink('huawei')} target='_blank' rel='noopener noreferrer'>
-                            <Icon icon='IcInstallationHuawei' width={135} height={40} />
-                        </a>
-                    </div>
 
-                    <div className='cfd-trade-modal__download-center-options--qrcode cfd-trade-modal__download-center-options--qrcode-hide'>
-                        <img src={getUrlBase('/public/images/common/mt5_download.png')} width={80} height={80} />
-                        <Text align='center' size='xxs'>
-                            {localize('Scan the QR code to download {{ platform }}.', {
-                                platform: getPlatformSettings('mt5').name,
-                            })}
-                        </Text>
-                    </div>
-                </div>
+                {is_mobile ? (
```

Comment:
Pls do an early return for dis

Reply:
idts!

```JavaScript
if(is_mobile) {<MT5MobileRedirectOption mt5_trade_account={mt5_trade_account} />}
<MT5DesktopRedirectOption account_title={getAccountTitle()} mt5_trade_account={mt5_trade_account} />
```

in suggested solution `MT5DesktopRedirectOption` will still be rendered if `is_mobile` when it's supposed to be one **or** another not both